src/EdFi.AnalyticsMiddleTier.Tests/TestHarnessBase.cs

```diff
@@ -206,4 +206,4 @@ public virtual IDbConnection OpenConnection()
             throw new NotImplementedException();
         }
     }
-}
```

Comment:
Above we have an interesting example of a "suggestion" or "note" that did not stop the build process. Here's a screenshot in case someone dismisses the alert. There doesn't seem to be a way to see this [at the command line](https://stackoverflow.com/questions/68976439/how-to-get-c-sharp-analyzer-suggestions-visible-on-the-commandline), though you can find the message in the sarif log file. You will also see this message in the build output in Visual Studio.

Reply:
![image](https://user-images.githubusercontent.com/9324390/178806328-10be2564-bb61-4bee-bcf8-2cd78ab79b33.png)